spec/features/users_sees_categories_items_spec.rb

```diff
@@ -0,0 +1,50 @@
+require "rails_helper"
+
+RSpec.feature "Users can see category items" do
+  scenario "page displays items in category" do
+    # Given that category exist
+    # And items are assigned
+    # As a user
+    # I visit '/category_name'
+    # I see the name of the category
+    # I also see all the items that belongs to that category
+    category = Category.create(name: "grocery")
+    category.items.create(title: "Banana",
+                          description: "Wholesome Yellow Goodness",
+                          price: 19,
+                          image_path: "http://saltmarshrunning.com/wp-content/uploads/2014/09/bananasf.jpg")
+
+    visit "/grocery"
+
+    expect(page).to have_content("Grocery")
+    expect(page).to have_content("Banana")
+  end
+
+  scenario "page displays items in category" do
+    # Given that category exist
+    # And items are assigned
+    # As a user
+    # I visit '/category_name'
+    # I see the name of the category
+    # I also see all the items that belongs ONLY to that category
```

Comment:
I like the sad path testing, it ensures we aren't seeing _all_ the items, only category items